src/seedu/addressbook/commands/Command.java

```diff
@@ -1,24 +1,42 @@
 package seedu.addressbook.commands;
 
-import seedu.addressbook.TextUi;
+import seedu.addressbook.common.Messages;
 import seedu.addressbook.model.AddressBook;
+import seedu.addressbook.model.person.ReadOnlyPerson;
+
+import java.util.List;
 
 /**
  * Represents a command with hidden internal logic and the ability to be executed.
  */
-public interface Command {
+public abstract class Command {
```

Comment:
Converted to a class because that seems like the better choice. But I'll think of how to remove the `abstract` as it is not covered yet.